contracts/contracts/vault/VaultCore.sol

```diff
@@ -212,33 +213,9 @@ contract VaultCore is VaultStorage {
      * @dev Allocate unallocated funds on Vault to strategies.
      **/
     function _allocate() internal {
-        // Harvest for all reward tokens above reward liquidation threshold.
         // Harvesting is the first action that takes place so we have an up to
         // date picture of total assets before allocating to strategies.
-        for (uint256 i = 0; i < allStrategies.length; i++) {
-            IStrategy strategy = IStrategy(allStrategies[i]);
-            address rewardTokenAddress = strategy.rewardTokenAddress();
-            if (rewardTokenAddress != address(0)) {
-                uint256 liquidationThreshold = strategy
-                    .rewardLiquidationThreshold();
-                if (liquidationThreshold == 0) {
-                    // No threshold set, always harvest from strategy
-                    IVault(address(this)).harvestAndSwap(allStrategies[i]);
-                } else {
-                    // Check balance against liquidation threshold
-                    // Note some strategies don't hold the reward token balance
-                    // on their contract so the liquidation threshold should be
-                    // set to 0
-                    IERC20 rewardToken = IERC20(rewardTokenAddress);
-                    uint256 rewardTokenAmount = rewardToken.balanceOf(
-                        allStrategies[i]
-                    );
-                    if (rewardTokenAmount >= liquidationThreshold) {
-                        IVault(address(this)).harvestAndSwap(allStrategies[i]);
-                    }
-                }
-            }
-        }
+        IHarvester(harvesterAddress).harvestAndSwap();
```

Comment:
As discussed in Harvester.sol, we can drop the harvestAndSwap, as well as the import above.

Reply:
Yup removed 👍